platform/default/mbgl/storage/offline_database.cpp

```diff
@@ -50,7 +50,10 @@ void OfflineDatabase::ensureSchema() {
             case 2: migrateToVersion3(); // fall through
             case 3: // no-op and fall through
             case 4: migrateToVersion5(); // fall through
-            case 5: return;
+            case 5:
```

Comment:
To do this right you need to change the version number,  `db->exec("PRAGMA user_version = 5");`, and add a `case 6`, here, but if your not expecting this to get merged upstream, then this is probably the right thing to do for now.